astroplan/utils.py

```diff
@@ -1,10 +1,118 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+import warnings
+import numpy as np
+from astropy.utils.data import download_file, clear_download_cache
+from astropy.utils import iers
+from astropy.time import Time
+import astropy.units as u
+from .exceptions import OldEarthOrientationDataWarning
+from astropy.utils.data import (_get_download_cache_locs, CacheMissingWarning,
+                                _open_shelve)
 
-__all__ = ["_mock_remote_data"]
+__all__ = ["_mock_remote_data", "get_IERS_A_or_workaround", "download_IERS_A"]
```

Comment:
Remove `_mock_remote_data` from `__all__` because it's not part of the public API?